### PR TITLE
Prepare to Go Live w/ Fern

### PR DESCRIPTION
**Pull request by noanflaherty**
## Changes
1. Updated readme
2. Updated spec with a freshly generated one from our monorepo. I manually added servers and x-request-name + confirmed `fern check` passes locally.
3. Remove unneeded secrets.

## Next Steps
1. Create release 0.21 in this github repo to trigger a new release to npm and pypi
2. Pull down latest versions and test manually e2e
3. Update README of this repo to explain how to perform a release
4. Update READMEs of client repos to mention correct npm and pypi package names
5. Update code snippets in Vellum UI
6. Announce to customers

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -29,13 +29,9 @@ jobs:
       - name: Download Fern
         run: npm install -g fern-api
 
-      - name: Release SDKs, OpenAPI, Postman
+      - name: Release SDKs
         env:
           FERN_NPM_TOKEN: ${{ secrets.FERN_NPM_TOKEN }}
```
Comment (dsinghvi) [inline .github/workflows/ci.yml:34]: could see you renaming this to NPM_TOKEN

File: fern/api/openapi/openapi.yml [hunk 2]
```diff
@@ -295,11 +295,21 @@ paths:
               schema:
                 $ref: '#/components/schemas/UploadDocumentErrorResponse'
           description: ''
-      servers:
-        - url: https://documents.vellum.ai
-          x-name: Documents
```
Comment (dsinghvi) [inline fern/api/openapi/openapi.yml:300]: was this intentional? looks like this endpoint is still served at `https://documents.vellum.ai`

Comment (noanflaherty) [reply fern/api/openapi/openapi.yml:300]: Nope not intentional - thanks for catching!

File: fern/api/generators.yml [hunk 3]
```diff
@@ -5,19 +5,19 @@ groups:
         version: 0.5.15-rc1
         output:
           location: npm
-          package-name: '@fern-api/vellum'
+          package-name: 'vellum-client'
           token: ${FERN_NPM_TOKEN}
```
Comment (dsinghvi) [inline fern/api/generators.yml:9]: if you update to `NPM_TOKEN`, you'd have to modify this too

**Pull request merged**